<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>кружочки</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addEditButton">
      <property name="text">
       <string>Поместить/Обновить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameLineEdit">
      <property name="placeholderText">
       <string>Кофе</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="descriptionTextEdit">
      <property name="placeholderText">
       <string>Описание</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="priceLineEdit">
      <property name="placeholderText">
       <string>Прайс</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>